security/bin/e2e.sh

```diff
@@ -50,8 +50,32 @@ if [[ "$HUB" =~ ^gcr\.io ]]; then
   gcloud docker --authorize-only
 fi
 
-# Build and push the Istio-CA docker image
-bin/push-docker.sh -h $HUB -t $TAG
+OUTPUT_DIR=`pwd`/docker
+
+# Generate certificate and private key from root
+echo 'Generate certificate and private key from root'
+bazel run $BAZEL_ARGS //security/cmd/generate_cert -- \
+-out-cert=${OUTPUT_DIR}/istio_ca.crt \
+-out-priv=${OUTPUT_DIR}/istio_ca.key \
+-organization="k8s.cluster.local" \
+-self-signed=true \
+-ca=true
+
+# Generate certificate and private key from istio_ca
```

Comment:
'Generate certificate and private key for node agent'. You may want to add echo here, too :)

Reply:
Done